src/main/java/exercise31/KarvonenHeartRate.java

```diff
@@ -13,19 +14,24 @@ public int formula(int age, int restingHR, double intensity) {
         return bpm;
     }
 
+    //Naming / data-type?
     public String[] checkDifferentIntensities(int age, int restingHR) {
```

Comment:
Is `checkDifferentIntensities` the best name, not sure it really describes what it does?  Also it's returning what's really an array of `int` as an array of `String` - as before.